lib/plurimath/math/formula/mrow.rb

```diff
@@ -0,0 +1,159 @@
+# frozen_string_literal: true
+
+module Plurimath
+  module Math
+    class Formula
+      class Mrow < Formula
```

Comment:
Is this `mrow` (and `mstyle`) a concept corresponding to MathML or is this to represent MathML? i.e. is this a generic concept? If it's a generic concept, should we just call it `row` and `style`?

Reply:
@ronaldtse, no, they are not generic and exist only for **MathML** parsing using **LutaML-Model**.